scripts/txt2img.py

```diff
@@ -79,23 +79,33 @@ def main(args):
 
     pipe = __create_inference_pipe(unet, text_encoder, tokenizer, sample_scheduler, vae)
 
-    seed = args.seed if args.seed != -1 else random.randint(0, 2**30)
-    gen = torch.Generator(device="cuda").manual_seed(seed)
-
-    img = __generate_sample(pipe, args.prompt, args.cfg_scale, args.resolution, gen=gen, steps=args.steps)
+    for _ in range(args.batch_count):
+        seed = args.seed if args.seed != -1 else random.randint(0, 2 ** 30)
+        gen = torch.Generator(device="cuda").manual_seed(seed)
 
+        images = __generate_sample(pipe, args.prompt, args.cfg_scale, args.height, args.width, gen=gen,
+                                   steps=args.steps,
+                                   batch_size=args.batch_size)
 
-    img.save(f"img_{args.prompt[0:100].replace(' ', '_')}_cfg_{args.cfg_scale}.png")
+        # img.save(f"output/img_{args.prompt[0:100].replace(' ', '_')}_cfg_{args.cfg_scale}.png")
+        timestamp = datetime.now().strftime("%Y%m%d-%H%M%S")
+        for i, img in enumerate(images):
+            img.save(
+                f"output/img_{args.prompt[0:210].replace(' ', '_')}_cfg_{args.cfg_scale}_{i}_{seed}_{timestamp}.png")
```

Comment:
This assume a max length of 255 for filenames, dunno if you think that's too risky.